db/seeds.rb

```diff
@@ -33,4 +33,4 @@
 
 
 #PetApplications
-@pet_applications_1 = PetApplication.create!(pet_id: "#{@pet_1.id}", application_id: "#{@cory.id}", status: "Pending" )
```

Comment:
Yeah we needed this at first but now that we can create the PetApplications through the website, it's better to test i there rather than seed out database with it.